tests/test_tree.py

```diff
@@ -53,6 +57,44 @@ def test_unserialized_tree_same_as_serialized(self):
         self.maxDiff = None
         self.assertEqual(before_load, after_load)
 
+        # leave the tree in a clean state
+        tree.clear()
```

Comment:
The API manager is recreated at the beginning of every single test, no need to clear it when it ends.